arbitrum-docs/run-arbitrum-node/06-troubleshooting.mdx

```diff
@@ -144,98 +144,260 @@ If you're running into unexpected outputs or errors, the following checklist may
+          </td>
+          <td>
+            This is because your node db got crashed and lost some message, you can try set
+            `--init.reorg-to-message-batch x-1`
+          </td>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see{' '}
+            <code>`Failed to load snapshot` err="head doesn't match snapshot: have a, want b”</code>
+          </td>
+          <td>
+            This is usually because an ungraceful shutdown causes a corrupted database; try
+            restarting the node without a prune flag, and after your node goes back to normal, then
+            graceful shutdown it and restart to prune it.
+          </td>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see{' '}
+            <code>
+              failed to get blobs: expected at least six blobs for slot `[slot_number]` but only got
+              0
+            </code>
+          </td>
+          <td>
+            This often happens when you connect to a beacon chain endpoint while the blob you are
+            querying is expired. To resolve this error, connect to a beacon endpoint which supports
+            historical blob data (see{' '}
+            <a href="/run-arbitrum-node/l1-ethereum-beacon-chain-rpc-providers#list-of-ethereum-beacon-chain-rpc-providers">
+              List of Ethereum beacon chain RPC providers
+            </a>
+            ).
+          </td>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see <code>P2P server will be useless, neither dialing nor listening</code>
+          </td>
+          <td>Arbitrum Nitro doesn’t need P2P mode, so you can ignore this log.</td>
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+  <TabItem value="error">
+    <table className="small-table" id="troubleshooting-table-warn">
+      <tbody>
+        <tr>
+          <th style={{ minWidth: 180 + 'px' }}>Scenario</th>
+          <th>Solution</th>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see <code>no contract code at given address</code>
+          </td>
+          <td>
+            Your parent chain node might not sync to the latest state, please wait after it finishes
+            syncing.
+          </td>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see{' '}
+            <code>
+              `staker: error checking latest staked` err="latest assertion of x: globalstate not in
+              chain: count a hash b expected c, sendroot d expected f"
+            </code>
+          </td>
+          <td>
+            Once it catches up, the node will check the state against the latest confirmed assertion
+            staked on chain; if it doesn’t match, it will log this error. Usually, this is because
+            of db corruption, so you might need to re-sync the blockchain; using a snapshot might
+            help: https://snapshot-explorer.arbitrum.io/
```

Comment:
```suggestion
            Once it catches up, the node will check the state against the latest confirmed assertion
            staked onchain; if it doesn’t match, it will log this error. Usually, this is because
            of db corruption, so you might need to re-sync the blockchain; using a snapshot might
            help: https://snapshot-explorer.arbitrum.io/
```